### DISCUSSION TITLE
404 Page not found

### DISCUSSION THREAD

**Discussion by gurkansoydan** [annotorious/annotorious#448] [General]
Hello, the url https://annotorious.dev/react/components-reference/#imageannotationpopup is  going to 404 page,  i need example popup code

**Comment by rsimon** [comment]
Sorry, I didn't have the time yet to write the documentation. If it helps: here's a minimal example for using the React `ImageAnnotationPopup` component:

https://github.com/annotorious/annotorious/blob/main/packages/annotorious-react/test/image/App.tsx
